### PR TITLE
Add schema and reportdb migration paths for 5.0

### PR DESCRIPTION
**Pull request by raulillo82**
## What does this PR change?

Add schema and reportdb migration paths for 5.0

## GUI diff

No difference.

Before:

After:

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: already covered

- [X] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/22011

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: schema/reportdb/uyuni-reportdb-schema.changes.raul.schema_reportdb_migration_paths_5.0 [hunk 1]
```diff
@@ -0,0 +1 @@
+- Bump version to 5.0.2
```
Comment (deneb-alpha) [inline schema/reportdb/uyuni-reportdb-schema.changes.raul.schema_reportdb_migration_paths_5.0:1]:
```suggestion
- Add missing migration paths to version 5.0.2
```

File: schema/spacewalk/susemanager-schema.changes.raul.schema_reportdb_migration_paths_5.0 [hunk 2]
```diff
@@ -0,0 +1 @@
+- Bump version to 5.0.2
```
Comment (deneb-alpha) [inline schema/spacewalk/susemanager-schema.changes.raul.schema_reportdb_migration_paths_5.0:1]:
```suggestion
- Add missing migration paths to version 5.0.2
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
I would rephrase the changelogs a bit

**Review by mcalmer — APPROVED** [review]
LGTM

**Review by deneb-alpha — APPROVED** [review]
LGTM

**Review by juliogonzalez — APPROVED** [review]
